contracts/MechMarketplace.sol

```diff
@@ -570,20 +697,20 @@ contract MechMarketplace is IErrorsMarketplace {
     /// @param deliveryRates Corresponding set of actual charged delivery rates for each request.
     /// @param deliveryDatas Corresponding set of self-descriptive opaque delivery data-blobs.
     /// @param paymentData Additional payment-related request data, if applicable.
-    function deliverMarketplaceBatchWithSignatures(
+    function deliverMarketplaceWithSignatures(
```

Comment:
This one is not modified, except for naming. Refer to the deliver with signatures PR.